### PR TITLE
feat: add Strategy[StrategymUSDCurve]

### PR DESCRIPTION
**Pull request by orbxball**

### CODE REVIEW

File: contracts/strategies/StrategymUSDCurve.sol [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    address public constant mpool = address(0x8474DdbE98F5aA3179B3B3F5942D724aFcdec9f6);
+    address public constant m3crv = address(0x1AEf73d49Dedc4b1778d0706583995958Dc862e6);
+    address public constant yvm3crv = address(0x0FCDAeDFb8A7DfDa2e9838564c5A1665d856AFDF);
+
+    address public governance;
+    address public controller;
+    address public strategist;
+
+    uint256 public constant DENOMINATOR = 10000;
+    uint256 public treasuryFee = 1000;
```
Comment (storming0x) [inline contracts/strategies/StrategymUSDCurve.sol:48]: is this difference from the other strats expected?

https://github.com/iearn-finance/yearn-protocol/blob/2252c4f1f2af464dbf066da8e41a57498e4f6330/contracts/strategies/StrategyUSDC3pool.sol#L48

File: contracts/strategies/StrategymUSDCurve.sol [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    uint256 public withdrawalFee = 0;
+    uint256 public strategistReward = 1000;
```
Comment (storming0x) [inline contracts/strategies/StrategymUSDCurve.sol:50]: here as well

https://github.com/iearn-finance/yearn-protocol/blob/2252c4f1f2af464dbf066da8e41a57498e4f6330/contracts/strategies/StrategyUSDC3pool.sol#L50

**Review by storming0x — APPROVED** [review]
LGTM! ran the diff against the following previous strats

[StrategyUSDC3pool](https://github.com/iearn-finance/yearn-protocol/blob/develop/contracts/strategies/StrategyUSDC3pool.sol)
[StrategyUSDT3pool](https://github.com/iearn-finance/yearn-protocol/blob/develop/contracts/strategies/StrategyUSDT3pool.sol)
[StrategyTUSDypool](https://github.com/iearn-finance/yearn-protocol/blob/develop/contracts/strategies/StrategyTUSDypool.sol)

File: contracts/strategies/StrategymUSDCurve.sol (hunk 1 above)
Comment (orbxball) [reply contracts/strategies/StrategymUSDCurve.sol:48]: Yes, it uses v2 params with v1 structure. This was about to be released on v2, but Andre said putting it on v1 was ideal.

**Review by fubuloubu — APPROVED** [review]
I don't see anything at issue

**Pull request merged**